pyfar/dsp/dsp.py

```diff
@@ -312,6 +312,263 @@ def spectrogram(signal, window='hann', window_length=1024,
     return frequencies, times, spectrogram
 
 
+def normalize(signal, normalize='time', normalize_to='max',
```

Comment:
I would suggest to rename `normalize` to `mode` and `normalize_to` to `normalize`